server/index.js

```diff
@@ -70,6 +71,22 @@ const createApp = () => {
 
   // static file-serving middleware
   app.use(express.static(path.join(__dirname, '..', 'public')))
+  app.use(require('body-parser').text())
+
+  app.post('/charge', async (req, res) => {
+    try {
+      let {status} = await stripe.charges.create({
+        amount: 2000,
+        currency: 'usd',
+        description: 'An example charge',
+        source: req.body
+      })
+
+      res.json({status})
+    } catch (err) {
+      res.status(500).end()
```

Comment:
I think we could create a new `err` here and give it the status 500 and a new error message, but then we should let the error handling middleware handle sending it out. so instead of sending out the status right away, just using `next(err)`